### ISSUE TITLE
use too much memory at quadview examples

### ISSUE THREAD

**Issue by hurryhuang1007** [FNNDSC/ami#294]
It seems to create three StackHelper object when implement [quadview examples](https://github.com/FNNDSC/ami/blob/dev/examples/viewers_quadview/viewers_quadview.js), and the memory three times the same also.

Is there a better way to do this?

**Comment by NicolasRannou** [comment]
Yes, the example was never meant to be used as reference but more of quick ways to go. That should be cleaned up to provide reference implementations...
Ideally, you should just use 1 material and share it across slices.

**Comment by hurryhuang1007** [comment]
I'm confused,the example create three StackHelper object because it has three view.
And I'm looking for a way,which is three view share one StackHelper object to achieve reduce the use of memory.

If that doesn't work, does that mean that I have to create 'StackHelper' myself? And create three SliceHelper by this 'StackHelper'.

**Comment by hurryhuang1007** [comment]
Thank you very much for your answer.

Is it means will use three times dedicated GPU memory?

I'm loading an about 150MB DICOM data to single view(just axial), the memory rise about 500MB, and the dedicated GPU memory rise about 0.2GB.
But I'm loading the 150MB data to three view(axial, sagittal and coronal), the memory rise about 500MB, the dedicated GPU memory rise about 0.8GB.
The 0.8GB is horrible, I want know how reduce the footprint of this part of memory.

**Comment by eduardor2k** [comment]
i'm having the same problem, loading 5 dicom files, with 4 views. the browser consumes more than 4GB of memory
